display/tab.go

```diff
@@ -158,26 +159,30 @@ func makeNewTab() *tab {
 				}
 			}
 			t.page.Column++
+			t.view.ScrollTo(t.page.Row, t.page.Column)
```

Comment:
None of these added lines should be needed, please test without them. The actual scroll of the `TextView` is changed in `applyHorizontalScroll` and `applyScroll`.

Reply:
why aren't we using `applyScroll` instead of `applyHorizontalScroll` here https://github.com/makeworld-the-better-one/amfora/blob/d4c7e87838d2ad2cc58d3b56995ee456a614e238/display/tab.go#L188

Reply:
See the comments at the beginning of the section: https://github.com/makeworld-the-better-one/amfora/blob/d4c7e87838d2ad2cc58d3b56995ee456a614e238/display/tab.go#L126-L127

Whenever vertical scrolling happens, the keys are returned and the actual scrolling is handled by cview. If that needs to be changed it can, but it shouldn't matter for this PR.

Reply:
But this will cause issues when the user will change the `moveup` and `movedown` key-binds to something other than `k` and `j`. In that case cview won't handle the scrolling, it should be done manually by calling `applyScroll` and increment/decrement the value of `t.Page.Row`
One more issue is that `t.Page.Row` starts from 0 which means that even if we reach at the end of the page the value of `t.page.Row` can still increment.
This can be resolved by using `termH` and doing some mathematical operations to make sure that we don't overflow the value of `t.page.Row` beyond the `height`.

Reply:
Yes, you're correct. Those changes all sound good to me. That should be implemented in this PR. It means the `return`s will have to changed to `return nil` instead of `return event`, so the `TextView` (`t.view`) doesn't see the up and down keys.

Reply:
@makeworld-the-better-one There's one more problem, the help page doesn't implement any custom scrolling it depends upon the scrolling from`cview`. Should I implement that as well in this PR?

Reply:
@makeworld-the-better-one any update on what should I do about the help page?

Reply:
> But this will cause issues when the user will change the `moveup` and `movedown` key-binds to something other than `k` and `j`. In that case cview won't handle the scrolling, it should be done manually

Ok I actually figured out a solution to this, and it applies to help page too. cview has defaults for scrolling keys, but these can be overridden. See [here](https://docs.rocketnine.space/code.rocketnine.space/tslocum/cview#pkg-variables) and scroll down to `var Keys =`.

So basically what should be done is handle left/right scrolling keys here. For up/down scrolling, set the keys in the `cview.Keys` map before starting, like in the `config/` package. You should also set the left/right scroll keys, because despite them being captured here, they're needed for the help page as you pointed out.

Does that make sense? Thanks for waiting.

Reply:
Seems like a good solution. I have implemented this you can review that and let me know if they are in correct place or not.

Reply:
Looks good!